package.json

```diff
@@ -19,7 +19,7 @@
     "eslint": "lerna exec --stream --no-bail -- eslint --max-warnings=0 src/**/*.ts",
     "start": "yarn -s tinymce-grunt start",
     "dev": "npm-run-all oxide-icons-build oxide-build -p tsc \"tinymce-grunt dev\"",
-    "ci": "npm-run-all -p oxide-ci oxide-icons-ci -p tsc \"tinymce-grunt dev\"",
+    "ci": "npm-run-all strict-type-check -p oxide-ci oxide-icons-ci -p tsc \"tinymce-grunt dev\"",
```

Comment:
This will run on actual browser nodes as well. If you might recall, in the call today @TheSpyder mentioned it should really only run on the compile node. That means this needs to be in `ci-all` instead of `ci`. It also means we might need to add it to `local-ci` as well.

Reply:
@tiny-ben-tran with the latest commit that means it's no longer running locally when running `yarn test`. As I mentioned above, you'll likely need to add it to `local-ci` as well to fix that.

Reply:
Don't need to add the strict type check to `local-ci` as it already calls `ci-all`

Reply:
Ahh yeah you're right sorry 🤦